.github/workflows/_testacc_platform.yml

```diff
@@ -0,0 +1,106 @@
+name: TestAcc Platform
+
+on:
+  workflow_call:
+  workflow_dispatch:
+  pull_request:
+    # Most Platform resources still use SDKv2 so we have to target files manually until migrated
+    paths:
```

Comment:
This PR looks fine. The only part I worry about is us adding in some new file and forgetting to enumerate it here. We'd probably think the tests ran fine. Any way you know of to prevent that?

Reply:
Yep, this will be resolved once we move to Plugin Framework for the Platform resources. In that case they will have their own go package and folder which can stay the same.

Reply:
The other thing to note is this is only for pull request convenience. The gate for shipping a release will use workflow call and has no filter for paths.